doc/specs/vulkan/appendices/glossary.txt

```diff
@@ -1021,8 +1021,9 @@ Render Pass Instance::
     A use of a render pass in a command buffer.
 
 Required Extensions::
-    Extensions which must: be enabled to use a specific enabled extension
-    (see <<extended-functionality-extensions-dependencies, Extension
+    Extensions that must: be enabled to be able to enable specific
+    extension dependent on them (see
```

Comment:
I think it will read more clearly if you change this sentence to something like:

> Extensions that must: be enabled in order to enable another extension that is dependent on them (see ...

Reply:
Both sound horrible to me...  :/

Reply:
Yes, all the forms of this language are pretty awkward.  I don't really like either update suggestion, nor the original.  Perhaps it warrants a more thorough rewording?

Reply:
Hmm.  Looking at this discussion, and below, I agree with James.  The "required extensions" term seems wrong, because it's defined narrowly here, and yet a looser-defined version of the term gets thrown around internally (e.g. the goal of making it clear which core version and/or extension is required in order to use a command, struct, etc.).  Bottom line (might not be saying it correctly), but I agree with James that we should step back and do a more thorough rewording.